src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Set-AzsComputeQuota.ps1

```diff
@@ -143,16 +157,32 @@ function Set-AzsComputeQuota {
                     $NewQuota = Get-AzsComputeQuota -Location $Location -Name $Name
                 }
 
+				$QuotaToBeSet = Convert-CustomQuotaToSDKObject -CustomQuota $NewQuota
+
                 # Update the Quota object from anything passed in
-                $flattenedParameters = @('AvailabilitySetCount', 'CoresLimit', 'VmScaleSetCount', 'VirtualMachineCount' )
+                $flattenedParameters = @('AvailabilitySetCount', 'CoresCount', 'VmScaleSetCount', 'VirtualMachineCount', 'StandardManagedDiskAndSnapshotSize', 'PremiumManagedDiskAndSnapshotSize' )
                 $flattenedParameters | ForEach-Object {
                     if ($PSBoundParameters.ContainsKey($_)) {
-                        $NewQuota.$($_) = $PSBoundParameters[$_]
```

Comment:
Delete new stuff, restore old code, and change $NewQuota to be $QuotaToBeSet

Reply:
done changes, not sure if that is what u meant. bit confusing.